n-n_pipeline.py

```diff
@@ -0,0 +1,45 @@
+'''
```

Comment:
Python filenames typically use snakecase, e.g. `this_is_a_file.py`. I think a name like `nn_pipeline.py` or `n_n_pipeline.py` would be better.